benchmark/build.sh

```diff
@@ -7,7 +7,7 @@ pushd tpcds-kit/tools
 make
 popd
 BUILD_DIR=build
-mkdir ${BUILD_DIR} || true
+mkdir -p ${BUILD_DIR} || true
```

Comment:
Why is the -p needed? Seems like the parent directory (this directory) is guaranteed to exist.

Reply:
I used -p to suppress "already exists" warning